Gemfile

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'byebug'
+gem 'pry-byebug'
+
+group 'deployment' do
```

Comment:
why do we need a group?

Reply:
meh, probably don't.  Just doing the normal thing.